### ISSUE TITLE
Shrink-to-fit width calculation off in specific scenario

### ISSUE THREAD

**Issue by bsweeney** [dompdf/dompdf#2677]
Dompdf appears to calculate a slightly smaller container width than required to fit content in certain scenarios.

The scenario I've found is when a positioned element contains another element and a the contained text has curly braces styled larger. And only if the text falls within a certain width.

The following appears to trigger the miscalculation:

```
<div style="position: fixed; background-color: yellow;">
    <div>lorem ipsum <span style="font-size: 120%; font-weight: bold; font-style: italic;">{}</span></div>
</div>
```

**Comment by Mellthas** [comment]
This is likely the same issue as #2560 and #2755, though I am not able to reproduce the issue here.

**Comment by Mellthas** [comment]
Okay, I can reproduce this with a font size of 125%. As it is fixed by #2761, I think we can consider this a duplicate of #2560.
